frontend/src/app/pages/exercises/exercise/shared/exercise-map/shared/cater-capacity/cater-capacity.component.html

```diff
@@ -0,0 +1,29 @@
+<div *ngIf="canCaterFor" style="max-width: 25vw">
+    <span>{{
+        type === 'material' ? 'Dieses Material ' : 'Diese Einsatzkraft '
```

Comment:
Ein Material kann keinen Patienten helfen/versorgen, sondern dafür genutzt werden, also vllt sowas, wie:

Dieses Material kann zur Versorgung von Patienten verschiedener Verletzungsgrade ...

Reply:
I get your point, but then we would have to dispatch the whole sentence based on the type, at which point we could also move the sentence to the personnel and material popup and just keep the status badge counters in this component. I think we should wait for christians opinion for this choice of words.